### PR TITLE
Update pul_metadata_services.

### LINKED ISSUE #1: Upgrade to the latest version of pul_metadata_services
The API for pul_metadata_services has changed but pul-worthwhile was not updated to consume it.  For now, I've pinned to the old version of pul_metadata_services.  Update the gemfile to use the latest version and refactor the code as necessary.

### PR DESCRIPTION
**Pull request by terrellt**
Closes #1 

If we merge this we'll need another ticket to unmark the EAD spec as pending.

### CODE REVIEW

**Comment by jpstroop** [comment]
Had this in progress before I had to cut out. Yours is prettier.

File: app/models/scanned_book.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class ScannedBook < ActiveFedora::Base
   include ::CurationConcerns::BasicMetadata
   include ::NoidBehaviors
 
+  property :sort_title, predicate: ::RDF::URI.new("http://library.princeton.edu/terms/primary_title"), multiple: false
```
Comment (terrellt) [inline app/models/scanned_book.rb]: We might want to find a real predicate for this.

**Comment by jpstroop** [comment]
I'm :ok: with merging, but can't from my phone for some reason.

File: app/models/scanned_book.rb (hunk 1 above)
Comment (jpstroop) [inline app/models/scanned_book.rb]: I feel like I've looked for one before without luck or range/domain restrictions, but it's been a while.

It's one of those things that's far too practical for linked data :wink:

**Comment by terrellt** [comment]
SKOS.prefLabel gets close, but may not have the semantics of "this is a title'

**Comment by terrellt** [comment]
@jpstroop I've got a branch that puts sort_title in DCT.title and title in DCT.alternative. Is that accurate?

**Comment by jpstroop** [comment]
Other way around, I think. And where do all the other non-main entry titles
go (MARC records can have quite a few)? I still think we can need a
subproperty of dct:alternative that holds exactly one value which users do
not generally see but allows the UI to sort objects by title properly.

Nobody ever sees this title. It's just there so that

-Js

Sent via mobile. Please excuse typos, brevity, etc.

**Comment by terrellt** [comment]
Oh, right, I had this conversation with @scande3 a while back. A title to sort on because for some reason solr's processors don't do well enough. https://t.co/RMlXyXKq6W seems relevant (from @scande3)

**Comment by jpstroop** [comment]
Yup, exactly.

**Comment by jpstroop** [comment]
`mods:nonSort` and MARC 245 second indicator (which is informing what we're doing here) are functionally the same.

**Comment by terrellt** [comment]
@jpstroop Switched to using that group's recommended predicate (which doesn't exist yet, but it could!)

**Pull request merged**